translate/pb/pb_decoder_entry.go

```diff
@@ -10,46 +10,29 @@ import (
 	"google.golang.org/protobuf/types/dynamicpb"
 )
 
-// StdProtobufDecoder transforms the protocol buffer message to a tuple.
-// Sample code to call this method:
-// let sysl = //encoding.proto.decode(//encoding.proto.proto, //os.file('sysl.pb'));
-// let decodeSyslPb = //encoding.proto.decode(sysl);
-// let shop = decodeSyslPb('Module', //os.file("petshop.pb"));
-// petshop.apps("PetShopApi")
 var StdProtobufDecoder = rel.NewNativeFunction("decode", func(param rel.Value) (rel.Value, error) {
-	bytes, isBytes := param.(rel.Bytes)
-	if !isBytes {
-		return nil, fmt.Errorf("//encoding.proto.decode: param not bytes")
+	tuple, isTuple := param.(rel.Tuple)
+	if !isTuple {
+		return nil, fmt.Errorf("//encoding.proto.decode: param not tuple")
 	}
 
-	if bytes.Bytes() == nil {
-		// the call looks like `let sysl = //encoding.proto.decode(//encoding.proto.proto, //os.file('sysl.pb'))`
-		return rel.NewNativeFunction("decode$1", func(definition rel.Value) (rel.Value, error) {
-			_, is := tools.ValueAsBytes(definition)
-			if !is {
-				return nil, fmt.Errorf("//encoding.proto.decode: definition not bytes")
-			}
-
-			return definition, nil
-		}), nil
+	fdVal, found := tuple.Get(fileDescriptorSet)
```

Comment:
As before, this should be a tuple representation of a file descriptor set, not a tuple containing a file descriptor set. It's fine for now, but add a `// TODO` referencing an issue to address it at some point.